src/main/java/org/graphqlcrudjava/model/Attribute.java

```diff
@@ -0,0 +1,87 @@
+package org.graphqlcrudjava.model;
+
+import java.util.Objects;
+
+public class Attribute implements Comparable<Attribute> {
```

Comment:
I know I said use these `Attribute` and `Entity` models here, but can you cross-check with graphql-java based models here that you originally used in your other prototype. IMO it may be beneficial to use that library as it may have a query parsing logic already here we have nothing. If yes, then these can be replaced or be designed as extended classes of the graphql-java classes

Reply:
I will take a look on that.